packages/api/src/hooks/useSettings.ts

```diff
@@ -0,0 +1,17 @@
+import { useMemo } from 'react';
+import useFetch from '../useFetch';
+
+/** A custom hook to get user settings (email, date of birth, address etc) */
+const useSettings = () => {
```

Comment:
@farhan-nurzi-deriv This hook can be smarter by combining `get_settings` via `useFetch` and `set_settings` via `useRequest` and also make use of `useInvalidateQuery` 🙏🏻